debezium-connector-jdbc/src/test/java/io/debezium/connector/jdbc/CollectionNamingStrategyTest.java

```diff
@@ -28,60 +29,64 @@
 @Tag("UnitTests")
 public class CollectionNamingStrategyTest {
 
+    private static final SinkRecordFactory RECORD_FACTORY = new DebeziumSinkRecordFactory();
+
     @Test
     public void testDefaultTableNamingStrategy() {
         final JdbcSinkConnectorConfig config = new JdbcSinkConnectorConfig(Map.of());
-        final SinkRecordFactory factory = new DebeziumSinkRecordFactory();
         final DefaultCollectionNamingStrategy strategy = new DefaultCollectionNamingStrategy();
-        assertThat(strategy.resolveCollectionName(factory.createRecord("database.schema.table"), config.getCollectionNameFormat()))
+        assertThat(strategy.resolveCollectionName(RECORD_FACTORY.createRecord("database.schema.table"), config.getCollectionNameFormat()))
                 .isEqualTo("database_schema_table");
     }
 
     @Test
     public void testCollectionNamingStrategyWithTableNameFormat() {
         final JdbcSinkConnectorConfig config = new JdbcSinkConnectorConfig(Map.of(JdbcSinkConnectorConfig.COLLECTION_NAME_FORMAT, "kafka_${topic}"));
-        final SinkRecordFactory factory = new DebeziumSinkRecordFactory();
         final DefaultCollectionNamingStrategy strategy = new DefaultCollectionNamingStrategy();
-        assertThat(strategy.resolveCollectionName(factory.createRecord("database.schema.table"), config.getCollectionNameFormat()))
+        assertThat(strategy.resolveCollectionName(RECORD_FACTORY.createRecord("database.schema.table"), config.getCollectionNameFormat()))
                 .isEqualTo("kafka_database_schema_table");
     }
 
     @Test
     public void testDeprecatedTableNamingStrategyWithTableNameFormat() {
         final JdbcSinkConnectorConfig config = new JdbcSinkConnectorConfig(Map.of(JdbcSinkConnectorConfig.DEPRECATED_TABLE_NAME_FORMAT, "kafkadep_${topic}"));
-        final SinkRecordFactory factory = new DebeziumSinkRecordFactory();
         final DefaultCollectionNamingStrategy strategy = new DefaultCollectionNamingStrategy();
-        assertThat(strategy.resolveCollectionName(factory.createRecord("database.schema.table"), config.getCollectionNameFormat()))
+        assertThat(strategy.resolveCollectionName(RECORD_FACTORY.createRecord("database.schema.table"), config.getCollectionNameFormat()))
+                .isEqualTo("kafkadep_database_schema_table");
+    }
+
+    @Test
+    public void testDeprecatedDefaultTableNamingStrategyWithTableNameFormat() {
```

Comment:
@mfvitale You are right! I have added this and another test for checking the deprecated options work as expected! This test uses the deprecated option AND the deprecated value!